src/test/java/org/jabref/logic/importer/util/GrobidServiceTest.java

```diff
@@ -51,4 +64,11 @@ public void processInvalidCitationTest() {
         assertThrows(IOException.class, () -> grobidService.processCitation("iiiiiiiiiiiiiiiiiiiiiiii", GrobidService.ConsolidateCitations.WITH_METADATA));
     }
 
+    @Test
+    public void processPdfTest() throws IOException, ParseException, URISyntaxException {
+        Path file = Path.of(GrobidPdfMetadataImporterTest.class.getResource("LNCS-minimal.pdf").toURI());
+        List<BibEntry> response = grobidService.processPDF(file, importFormatPreferences);
+        assertEquals(1, response.size());
```

Comment:
I know, still WIP but would be better to compare a bib entry here instead of the sze

Reply:
True